src/map/packets/char_health.cpp

```diff
@@ -51,4 +51,27 @@ CCharHealthPacket::CCharHealthPacket(CCharEntity* PChar)
         ref<uint8>(0x22) = PChar->GetSJob();
         ref<uint8>(0x23) = PChar->GetSLevel();
     }
-}
\ No newline at end of file
+}
+// NPCFELLOW --------------------------------------------vv
+CCharHealthPacket::CCharHealthPacket(CFellowEntity* PFellow)
```

Comment:
this is the exact same as the other constructor, just change the parameter and use that one

Reply:
ummm... it looks like you did this already...